### DISCUSSION TITLE
How can I join readium slack ? I was part of the channel but cant get in anymore.

### DISCUSSION THREAD

**Discussion by mrifni** [readium/kotlin-toolkit#468] [Q&A] [answered]
How can I join readium slack ? I was part of the channel but cant get in anymore. thanks

**Comment by mrifni** [comment]
@mickael-menu

**Reply by mickael-menu** [reply, marked as answer]
The Readium Slack is not open anymore. You need to be [a member of the Readium Foundation](https://readium.org/membership/join/) or a significant contributor to get access.
